tests/test_clients_GcpClient.py

```diff
@@ -748,6 +751,47 @@ def test_download_from_blobstore_raises_exception_on_invalid_container(self):
             Exception, self.gcpClient._download_from_blobstore, 'blob', invalid_blob_path)
         self.gcpClient.container = prev_container
 
+    @patch('lib.clients.BaseClient.requests.post')
+    @patch('lib.clients.BaseClient.requests.get')
+    def test_gcp_client_creation(self,  mock_get, mock_post):
```

Comment:
Maybe rename this function to test_gcp_client_creation_with_cred_hub as the test result here only shows the function name, so will be more intuitive.

Reply:
ok